### PR TITLE
Update documentation with new features for 2.1

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | yes
| Translation   | yes
| Fixed tickets | 
| License       | MIT

Add documentation about article sharing

@wallabag/translators I need help to translate these new pages in german.

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ framework:
     assets: ~
 
 wallabag_core:
-    version: 2.0.7
+    version: 2.1
```
Comment (tcitworld) [inline app/config/config.yml]: This shouldn't be in this PR.

File: docs/fr/user/installation.rst [hunk 2]
```diff
@@ -35,25 +35,31 @@ Installation
...
 
 ::
 
     curl -s http://getcomposer.org/installer | php
 
-Vous pouvez trouver des instructions spécifiques `ici (en anglais) <https://getcomposer.org/doc/00-intro.md>`__ :
+Vous pouvez trouver des instructions spécifiques `ici (en anglais) <https://getcomposer.org/doc/00-intro.md>`__.
 
-Pour installer wallabag, vous devez exécuter ces deux commandes :
+Vous devez aussi installer node.js (via `la page de téléchargements <https://nodejs.org/en/download/>`__), npm (déjà installé avec node.js), bower et grunt:
```
Comment (tcitworld) [inline docs/fr/user/installation.rst]: No need to install nodejs, npm, bower & grunt, composer installs nodejs-installer then npm installs the rest of it. https://github.com/wallabag/wallabag/blob/2.1/install.sh

**Comment by nicosomb** [comment]
ping @Strubbl

**Comment by Quent-in** [comment]
Are there new strings to translate for the next release?

**Comment by j0k3r** [comment]
@Quent-in yep, they are 10 strings missing in OC: https://github.com/wallabag/wallabag/blob/2.1/src/Wallabag/CoreBundle/Resources/translations/messages.oc.yml

**Comment by nicosomb** [comment]
@Quent-in maybe the documentation too :) https://github.com/wallabag/wallabag/tree/2.1/docs

**Comment by nicosomb** [comment]
In fact, I updated all the documentation with 2.1.0 version.

File: README.md [hunk 3]
```diff
@@ -10,15 +10,14 @@ More informations on our website: [wallabag.org](https://wallabag.org)
...
+If you don't have it yet, please [install composer](https://getcomposer.org/download/) or be sure to use Composer 1.2 (`composer selfupdate` can help you about that). Read [our documentation](http://doc.wallabag.org) to have more information about installation. 
 Then you can install wallabag by executing the following commands:
 
 ```
     git clone https://github.com/wallabag/wallabag.git
     cd wallabag
-    git checkout 2.0.7
-    SYMFONY_ENV=prod composer install --no-dev -o --prefer-dist
-    php bin/console wallabag:install --env=prod
+    git checkout 2.1
```
Comment (j0k3r) [inline README.md]: 2.1.0

File: app/config/config.yml (hunk 1 above)
Comment (j0k3r) [inline app/config/config.yml]: 2.1.0 :)

**Comment by j0k3r** [comment]
:+1:

**Pull request merged**